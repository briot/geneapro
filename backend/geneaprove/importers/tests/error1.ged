0 HEAD
1 SOUR test for duplicate SUBM.NAME
1 CHAR UTF-8
1 GEDC
2 VERS 5.5.1
2 FORM LINEAGE-LINKED
1 SUBM @SUBM@
0 @SUBM@ SUBM
1 NAME Individual Tester
1 NAME Duplicate (and invalid) name
1 LANG English
0 @I1@ INDI
1 NAME Individual Middle /Tester1/
0 TRLR
