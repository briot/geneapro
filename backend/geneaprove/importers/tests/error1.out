OK
Line 10: Too many NAME in SUBM (skipped)
Line 11: Ignored SUBM.LANG (English)
